src/main.rs

```diff
@@ -32,6 +32,7 @@ use ctap2::api::connection::{HidConnection, SendOrRecvStatus};
 #[cfg(feature = "debug_ctap")]
 use ctap2::clock::CtapClock;
 use ctap2::clock::{new_clock, Clock, ClockInt, KEEPALIVE_DELAY, KEEPALIVE_DELAY_MS};
+use ctap2::ctap::hid::HidPacketIterator;
```

Comment:
This is the before mentioned reason we need `ctap` to be public right?

Reply:
Yes.